packages/core/admin/admin/src/hooks/useInjectReducer/useInjectReducer.js

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react';
+import { useStore } from 'react-redux';
+
+export function useInjectReducer(namespace, reducer) {
+  const store = useStore();
+
+  useEffect(() => {
+    store.injectReducer(namespace, reducer);
+  }, [store, namespace, reducer]);
```

Comment:
Not too familiar with the useStore hook, would this update with any store change or is this more the store object as opposed to the state store?

Reply:
I believe this the store object, because the docs at https://react-redux.js.org/api/hooks#usestore state: 

> This hook returns a reference to the same Redux store that was passed in to the <Provider> component.